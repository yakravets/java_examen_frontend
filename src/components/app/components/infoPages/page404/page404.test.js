import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
configure({ adapter: new Adapter() });
import toJson from "enzyme-to-json";

import Page404 from './page404';

it("renders correctly", () => {
    const tree = shallow( < Page404 /> );
    expect(toJson(tree)).toMatchSnapshot();
});

it("renders without crashing", () => {
    shallow( < Page404 /> );
});