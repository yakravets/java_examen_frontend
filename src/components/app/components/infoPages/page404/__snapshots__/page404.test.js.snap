// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <Result
    extra={
      <Button
        type="primary"
      >
        Back Home
      </Button>
    }
    status="404"
    subTitle="Sorry, the page you visited does not exist."
    title="404"
  />
</Fragment>
`;
